pkg/work/hub/controllers/manifestworkreplicasetcontroller/manifestworkreplicaset_deploy_test.go

```diff
@@ -222,7 +222,9 @@ func TestDeployWithRolloutStrategyReconcileAsExpected(t *testing.T) {
 	perGoupeRollOut := clusterv1alpha1.RolloutStrategy{
 		Type: clusterv1alpha1.ProgressivePerGroup,
 		ProgressivePerGroup: &clusterv1alpha1.RolloutProgressivePerGroup{
-			Timeout: clusterv1alpha1.Timeout{Timeout: "None"},
+			RolloutConfig: clusterv1alpha1.RolloutConfig{
+				Timeout: "None",
```

Comment:
This is a catch up of: https://github.com/open-cluster-management-io/api/pull/281